package/camera-control/camera-ctl

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
nit (you can change it or not): [The variables are not quoted](https://www.shellcheck.net/).